### DISCUSSION TITLE
ALOS ScanSAR data request

### DISCUSSION THREAD

**Discussion by KBodolai** [microsoft/PlanetaryComputer#327] [Data request]
In the last couple of years JAXA released the archive for [ALOS-2 PALSAR-2 ScanSAR data](https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm), processed as terrain-flattened $\gamma_0$ values (Level 2.2, and also Level 1.1 I think) 

It provides a time series of L-band data with a revisit time of ~42 days and about 100m of spatial resolution, I think it would be a great addition to the planetary computer data catalog.

Is there something in the works for ingesting this data? As far as I know some of it is available as COGs in AWS, but i don't think it's  STAC-indexed. 

Thanks!

**Comment by scottyhq** [comment]
Hi @KBodolai I don't think it's a complete or global release by JAXA but agreed this would be a nice dataset to have available!

Just wanted to note this effort to add a common STAC endpoint for AWS datasets, but as far as I can tell you're right that most AWS public geospatial datasets do not have STAC metadata yet https://developmentseed.org/blog/2023-10-20-asdi

**Reply by KBodolai** [reply]
Yes, as far as I can tell it covers most South America, Africa and SE Asia, so not global as you said, but pretty exciting! 

Thanks for sharing that, stactools-pipelines looks like an awesome effort indeed.
